src/rctmon/models.py

```diff
@@ -357,6 +359,11 @@ def collect_gen(gen: SolarGeneratorReadings, name: str, gen_name: str) -> None:
             ivs.add_metric([name], self.inverter_status)
             yield ivs
 
+        if self.inverter_grid_separated is not None:
+            igs = GaugeMetricFamily('rctmon_inverter_grid_separated', 'Status of the island mode', labels=['inverter', 'grid'])
```

Comment:
You're specifying that it requires two labels, but it only needs one (`inverter` name) and you're only giving it the one in the next line, so this should be `labels=['inverter']`.

Reply:
Thx, I removed grid label